src/AccessibilityInsights.SharedUx/FileBug/FileBugAction.cs

```diff
@@ -292,10 +292,7 @@ private static bool GuidsMatchInReproSteps(string targetGuid, string reproSteps)
         private static int? FileBugWindow(Uri url, bool onTop, int zoomLevel, Action<int> updateZoom)
         {
             System.Diagnostics.Trace.WriteLine(Invariant($"Url is {url.AbsoluteUri.Length} long: {url}"));
-            // To force latest IE to show up
-            var appName = System.Diagnostics.Process.GetCurrentProcess().ProcessName + ".exe";
-            using (var Key = Microsoft.Win32.Registry.CurrentUser.OpenSubKey(@"SOFTWARE\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BROWSER_EMULATION", true))
-                Key.SetValue(appName, 99999, Microsoft.Win32.RegistryValueKind.DWord);
+            BrowserEmulation.SetBrowserFeatureControl();
```

Comment:
>SetBrowserFeatureControl [](start = 29, length = 24)

Will this code need to move into the AzureDevOps extension?

Reply:
Yes, it will need to move. I will let @jalkire know about this change